pkg/kapp/cmd/appgroup/deploy.go

```diff
@@ -66,14 +66,14 @@ func (o *DeployOptions) Run() error {
 
 	// TODO what if app is renamed? currently it
 	// will have conflicting resources with new-named app
-	updatedApps, err := o.appsToUpdate()
+	updatedApps, sortedApps, err := o.appsToUpdate()
```

Comment:
I think we can just get rid of the map here, the only place where we seem to be using it is to delete apps that are no longer present, and I think we can just use a nested loop instead of the map to check for it's existence (I don't think that anyone is going to have thousands of directories to be created as apps so we don't need to worry much about time complexity here)

Reply:
Okay @praveenrewar i will get rid of  it.